doc/source/io.rst

```diff
@@ -4550,6 +4551,76 @@ Read from a feather file.
    import os
    os.remove('example.feather')
 
+
+.. _io.parquet:
+
+Parquet
+-------
+
+.. versionadded:: 0.21.0
+
+Parquet provides a sharded binary columnar serialization for data frames. It is designed to make reading and writing data
```

Comment:
IMO 'sharded' is too technical for this introduction (many users will already not understand this sentence). 

Can you also include a link to the parquet website?